consensus/ibft/snapshot.go

```diff
@@ -46,6 +46,28 @@ func (i *Ibft) setupSnapshot() error {
 		}
 	}
 
+	currentEpoch, metaEpoch := (header.Number / i.epochSize), (meta.LastBlock / i.epochSize)
```

Comment:
Please add comment to what we are doing here.

For example:
```
If the snapshot is not found, or the latest snapshot belongs to a previous epoch, we need to start processing headers from the beginning of the current epoch in order to have all the votes and validators correctly set in the snapshot, since they reset every epoch.
```

Reply:
Just a comment on the code style, open for discussion:

`x, y := 1, 2`
seems to be harder to read than
```
x := 1
y := 2
```